src/plot.js

```diff
@@ -35,7 +35,7 @@ export function plot(options = {}) {
       if (scale !== undefined) {
         const scaled = scaleChannels.get(scale);
         const {percent, transform = percent ? x => x * 100 : undefined} = options[scale] || {};
-        if (transform !== undefined) channel.value = Array.from(channel.value, transform);
+        if (typeof transform === "function") channel.value = Array.from(channel.value, transform);
```

Comment:
I didn’t mean we should silently ignore a invalid argument, like if you pass a transform object instead of a function. Instead I meant we should consider null and undefined to be equivalent here. I’ll make the change and merge.